src/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum AuctionFunction {
```

Comment:
There is already type `AuctionPricingFunction`

Reply:
fixed